mitodl/micromasters
# refactor: remove proctored exams card and FEATURE_TURN_PAYMENT_OFF flag

#### Pre-Flight checklist


#### What are the relevant tickets?
#5282 

#### What's this PR do?
- Refactored the code to remove `FEATURE_TURN_PAYMENT_OFF`
- Removed any conditions and the Proctored exam card from the dashboard

#### How should this be manually tested?
- Setup on this branch
- Load the dashboard and you should not see a `Proctored Exam` card on the dashboard
- Try to set `FEATURE_TURN_PAYMENT_OFF` to False and see that there should still be no proctored exam card
- As a matter of fact, adding `FEATURE_TURN_PAYMENT_OFF` in .env should not affect anything at all now
- Nothing should break


#### Where should the reviewer start?
Might be a good place to start testing by first enabling the `Proctored Exam` card as mentioned in the ticket by enabling `FEATURE_TURN_PAYMENT_OFF` so that it's visible on the dashboard, then shift to this branch

#### Any background context you want to provide?
We introduced this flag recently when we turned off the payments for MM, It was done in https://github.com/mitodl/micromasters/pull/5230/files

#### Screenshots (if appropriate)
**After refactoring and removal of `FEATURE_TURN_PAYMENT_OFF` flag**
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/34372316/213162256-5096f45d-fbd9-466e-a1bc-44f9465a423b.png">
